sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/ScalaReflection.scala

```diff
@@ -539,9 +541,10 @@ object ScalaReflection extends ScalaReflection {
 
         val params = getConstructorParameters(t)
         val fields = params.map { case (fieldName, fieldType) =>
-          if (javaKeywords.contains(fieldName)) {
-            throw new UnsupportedOperationException(s"`$fieldName` is a reserved keyword and " +
-              "cannot be used as field name\n" + walkedTypePath)
+          if (SourceVersion.isKeyword(fieldName) ||
+            !SourceVersion.isIdentifier(encodeFieldNameToIdentifier(fieldName))) {
```

Comment:
nit: one more indent.

Reply:
Is it needed?
There are some similar style found.

[JDBCRelation.scala](https://github.com/apache/spark/blob/051e691029c456fc2db5f229485d3fb8f5d0e84c/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCRelation.scala#L96)
[Utils.scala](https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/util/Utils.scala#L2391)
[CodeGenerator.scala](https://github.com/apache/spark/blob/112caea214ac354b771be11bbfd41cac2bf12a28/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeGenerator.scala#L1627)

Reply:
Yea, I know that. Not strong preference though, I found the style below in this file;
https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/ScalaReflection.scala#L283
```
if (XXXXX ||
    YYYY) { // one more indent to tell a difference from <code block>
  <code block>
}
```
So, I left the comment above for per-file style consistency. Anyway, trivial though.

Reply:
Hmm, exactly. O.K, I'll follow the rule. Thanks.